starter/pom.xml

```diff
@@ -135,6 +135,14 @@
                     </dependency>
                 </dependencies>
             </plugin>
+            <plugin>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <groupId>org.apache.maven.plugins</groupId>
+                <configuration>
+                    <source>8</source>
```

Comment:
Is the addition of this plugin needed? We can just upgrade this to java 17 right?

Reply:
No, that's a mistake. I suspect IntelliJ has done it's magic and added that. I've removed it from the pull request now.

If you like, we can keep
```
<maven.compiler.source>17</maven.compiler.source>
<maven.compiler.target>17</maven.compiler.target>
```
However, I think Spring adds that automatically.